### PR TITLE
bump friboo, add JAVA_OPTS, remove JSONWriter

### LINKED ISSUE #2: Update Friboo and allow to pass JVM options
- JVM: Because we want to set Hystrix timeouts
- Friboo: Because we're at it anyways.

### PR DESCRIPTION
**Pull request by prayerslayer**
Fixes #2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,6 +4,9 @@ MAINTAINER Zalando SE
 
 COPY target/essentials.jar /
 
-CMD java $(java-dynamic-memory-opts) $(appdynamics-agent) -jar /essentials.jar
+EXPOSE 8080
+ENV HTTP_PORT=8080
```
Comment (hjacobs) [inline Dockerfile]: why do we need this env var?

Comment (prayerslayer) [inline Dockerfile]: To be honest: I don't know, but it's also in our other Dockerfiles. I figured it has a purpose.

Comment (prayerslayer) [inline Dockerfile]: Probably it's to make sure that Friboo boots Jetty on 8080 and not something else.

Comment (prayerslayer) [inline Dockerfile]: Okay, no, it's useless. Will remove.

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**